src/aerie_cli/commands/command_context.py

```diff
@@ -0,0 +1,17 @@
+import typer
+
+app = typer.Typer()
+
+
+
+class CommandContext:
+    def __init__(self, hasura_admin_secret: str) -> None:
+        self._hasura_admin_secret = hasura_admin_secret
+    def get_hasura_admin_secret(self) -> str:
+        return self._hasura_admin_secret
+
+class CommandContextManager:
+    context: CommandContext = None
+    def __init__(self) -> None:
+        raise NotImplementedError
```

Comment:
Good point.

I thought the manager would help when testing to tell when the context was initialized versus when the values weren't being read from typer properly. This is mainly just to ensure that if we ever change how the context is set, issues with it are detected properly. The value of `_hasura_admin_secret` is allowed to be None, so just checking that would be insufficient. Having the manager would make it easy to tell that the value of `_hasura_admin_secret` wasn't set because the context itself was never created.

In retrospect, this is overkill and can still be tested fairly easily. There is not any need to make the context more complicated. It has been fixed with 96e5161